mlflow/utils/uri.py

```diff
@@ -72,6 +72,13 @@ def is_databricks_uri(uri):
     return scheme == "databricks" or uri == "databricks"
 
 
+def is_fuse_uri(uri):
+    """
+    Validates whether a provided URI is directed to a FUSE mount point
+    """
+    return any(uri.startswith(x) for x in [_DBFS_FUSE_PREFIX, _DBFS_HDFS_URI_PREFIX])
```

Comment:
We also have a Volumes path you will likely want to account for here: `/Volumes/`

Reply:
@BenWilson2